controllers/SiteController.php

```diff
@@ -61,7 +64,19 @@ public function actions()
      */
     public function actionIndex()
     {
-        return $this->render('index');
+
+        $executors = Users::find()
+            ->joinWith('cities')
+            ->where(['is_executor' => 1])
+            ->asArray()
+            ->all();
+
+        $category_names = [];
+        foreach (Categories::find()->all() as $category) {
+            $category_names[] .= $category->name;
+        }
+
+        return $this->render('index',['category_names' => $category_names, 'executors' => $executors]);
```

Comment:
добавить запятую для идеального кодестайла:
```suggestion
        return $this->render('index', ['category_names' => $category_names, 'executors' => $executors]);
```
😎